content/code_of_conduct.md

```diff
@@ -0,0 +1,119 @@
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+  
+Examples of unacceptable behavior include:
+
+  
+* Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, pregnancy status, veteran status, political affiliation, marital status, body size, age, race, national origin, ethnic origin, nationality, immigration status, language, religion or lack thereof, or other identity marker. This includes anti-Indigenous/Nativeness and anti-Blackness.
+* Unwelcome comments regarding a person's lifestyle choices and practices, including those related to food, health, parenting, relationships, drugs, and employment.
+* Deliberate misgendering, using inappropriate pronouns, or use of "dead" or rejected names.
+* Gratuitous or off-topic sexual images or behavior in spaces where they're not appropriate.
+* Physical contact and simulated physical contact (eg, textual descriptions like "hug" or "backrub") without consent or after a request to stop.
+* Threats of violence.
+* Incitement of violence towards any individual or group, including encouraging a person to commit suicide or to engage in self-harm.
+* Deliberate intimidation.
+* Stalking or following - online or in the physical world.
+* Harassing photography or recording, including logging online activity for harassment purposes.
+* Sustained disruption of discussion.
+* Unwelcome sexual attention.
+* Patterns of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others.
+* Continued one-on-one communication after requests to cease.
+* Deliberate "outing" of any aspect of a person's identity without their consent except as necessary to protect vulnerable people from intentional abuse.
+* Publication of non-harassing private communication, such as a physical or email address, without their explicit permission
+* Jokes that resemble the above, such as "hipster racism", still count as harassment even if meant satirically or ironically.
+* Trolling, insulting or derogatory comments, and personal or political attacks
+  
+
+If you have questions or concerns about these issues please feel free to contact [REQUIRES CONTACT POINTS] and share your opinions and suggestions.
+
+## Reporting
+
+
+If you are being harassed by a member of our community, notice that someone else is being harassed, or have any other concerns, please contact the administrators via  [REQUIRES CONTACT POINTS]. If the person who is harassing you is on the project collaborations team, they will not be involved in handling or resolving the incident.
```

Comment:
This could be me via Mattermost. Anyone else interested in being a point of contact?